### ISSUE TITLE
No longer works in Firefox 42

### ISSUE THREAD

**Issue by Gitoffthelawn** [clear-code/autopermission#2]
As of Firefox 42, this extension no longer works.

If more details are needed, just ask.

Is an update planned?

**Comment by piroor** [comment]
I've released the version 2.2, tested on Nightly 45.0a1.
It is now available on AMO website.

**Comment by Gitoffthelawn** [comment]
Thanks so much!

Unfortunately, I gave v2.2 a try in Firefox v42.0, and nothing shows up in about:permissions.

I also created a boolean named `extensions.autopermission.debug` and set it to `true`, but nothing new appears in the Browser Console.

**Comment by Gitoffthelawn** [comment]
Scratch that... it was an AMO bug.  Even though AMO was showing version 2.2, Firefox was downloading version 2.1.

I worked around that issue, and installed v2.2.

Version 2.2 works great so far!  Thank you!  Domo arigato!

Nothing is in the debug log, even with the debug pref set to true.  Is this expected, or is the debug feature not working?

**Comment by piroor** [comment]
You'll see debug logs in the Browser Console (Ctrl-Shift-J). They won't appear regular web console for the content area.

**Comment by Gitoffthelawn** [comment]
Right... but there are not entries in the Browser Console from this extension, even though the debug pref is enabled.  Perhaps this is just because everything is working properly?

**Comment by piroor** [comment]
`[autopermission] loadPolicies` and more logs are expected to appear in the console always. Hmm, something seems wrong on your environment...
